cosmos/providers/dbt/core/operators/local.py

```diff
@@ -125,6 +125,14 @@ def run_command(
                 tmp_project_dir,
             )
 
+            # ensure there's no profiles.yml file in the project dir
+            profiles_path = os.path.join(tmp_project_dir, "profiles.yml")
+            if os.path.exists(profiles_path):
+                logger.warning(
+                    "Cosmos doesn't support user-supplied profiles.yml files. Removing..."
+                )
+                os.remove(profiles_path)
```

Comment:
It's probably worth not deleting the `profiles_path` since users may be using it to build the Docker image for the docker or K8s modes.